src/clj_kondo/impl/analyzer/re_frame.clj

```diff
@@ -40,6 +41,20 @@
           (common/analyze-children ctx args)))
       (common/analyze-children ctx children))))
 
+(defn analyze-inject-cofx [ctx expr ns]
+  (let [kns (keyword ns)
+        [cofx-id cofx-param] (next (:children expr))]
+    (common/analyze-children (assoc-in ctx [:context kns :cofx-ref] true) [cofx-id])
+    (when cofx-param
+      (common/analyze-children ctx [cofx-param]))))
```

Comment:
I would just lint all the children from `(next (:children exor))` here, even though inject-cofx only supports two arguments, this will analyze all the arguments the user passes and will making linting more correct.

Reply:
Same for below

Reply:
pushed, hope i covered all cases